README.md

```diff
@@ -24,6 +24,26 @@
 * Mapstruct
 * Lombok
 
+## Start the application using docker
+* download the app
+$ git clone https://github.com/KSM-development/wst-backoffice.git
+
+* go to main app directory
+$ cd wst-backoffice
+
+* create a jar file
+$ mvn clean package
+
+* run the app in the docker
+$ docker-compose up
+
+* stop the app in the docker
+$ docker-compose down
+
+* test the app is up and running
+$ curl -X GET http://localhost:8081/addresses
```

Comment:
Please remove this, in the future, we will have millions of addresses, so we will change this endpoint to use pagination.

Reply:
done